### PR TITLE
Mino Mutual Scene

### PR DESCRIPTION
**Pull request by KiefKeith**
Added the Mino Mutual scene to the Minotaur Scene script. Added a button for this scene after the minotaur has been defeated, which checks if the player has a cock.
//Hopefully I did this right, never collaborated on github before.

### CODE REVIEW

File: classes/classes/Scenes/Areas/Mountain/MinotaurScene.as [hunk 1]
```diff
@@ -43,7 +44,7 @@ public function minoVictoryRapeChoices():void {
 	if (player.hasCock()) {
 		if (player.cockThatFits(80) >= 0) bj = minotaurBlowjob;
 	}
-	if (player.hasCock() && x >= 0) dickRape = bumRapeaMinotaur;
+	if (player.hasCock() && x >= 0) dickRape = bumRapeaMinotaur; mutual = minoMutual;
```
Comment (Stadler76) [inline classes/classes/Scenes/Areas/Mountain/MinotaurScene.as]: This should be
```as3
if (player.hasCock() && x >= 0) { dickRape = bumRapeaMinotaur; mutual = minoMutual; }
```
or better:
```as3
if (player.hasCock() && x >= 0) {
	dickRape = bumRapeaMinotaur;
	mutual = minoMutual;
}
```
or  it won't work as intended

Comment (KiefKeith) [inline classes/classes/Scenes/Areas/Mountain/MinotaurScene.as]: Ah, okay. Wasn't sure about that myself. Thanks.

**Comment by Stadler76** [comment]
Ok, found a couple parser bugs in paragraph 12 on page 2 if you're a herm.
Diff for that and the issue mentioned above:
```diff
diff --git a/classes/classes/Scenes/Areas/Mountain/MinotaurScene.as b/classes/classes/Scenes/Areas/Mountain/MinotaurScene.as
index 1f3ad547..fd17ccb6 100644
--- a/classes/classes/Scenes/Areas/Mountain/MinotaurScene.as
+++ b/classes/classes/Scenes/Areas/Mountain/MinotaurScene.as
@@ -44,7 +44,10 @@ public function minoVictoryRapeChoices():void {
 	if (player.hasCock()) {
 		if (player.cockThatFits(80) >= 0) bj = minotaurBlowjob;
 	}
-	if (player.hasCock() && x >= 0) dickRape = bumRapeaMinotaur; mutual = minoMutual;
+	if (player.hasCock() && x >= 0) {
+		dickRape = bumRapeaMinotaur;
+		mutual = minoMutual;
+	}
 	if (x >= 0 && player.cockTotal() > 1 && player.countCocksOfType(CockTypesEnum.TENTACLE) > 0) tentaRape = rapeMinotaurTentacles;
 	if (player.hasVagina()) cuntRape = girlRapeAMinotaur;
 	//Centaurs can't do the herm scene
@@ -1394,7 +1397,7 @@ private function minoMutualPt3(x:Number):void
 	}
 	else if (player.gender == 3) //Player is herm 
 	{
-		outputText("\n\nInstantly [eachCock] is at full mast as your [pussy] begs to be filled. The lust inducing mixture warms your [chest] the way alcohol would, probably due to the minotaur cum mixing with the lust draft. Your mind becomes consumed with the dual urges to fuck and be fucked. [EachCock] {strain / strains} for a tight hole to fill while your [pussy] wants something big and drooling stuffed into its drooling opening. Even now you feel the wetness of your drooling loins salivating and you canÔÇÖt help but wonder if there is another minotaur around here somewhere to make a [playername] sandwich.");
+		outputText("\n\nInstantly [eachCock] is at full mast as your [pussy] begs to be filled. The lust inducing mixture warms your [chest] the way alcohol would, probably due to the minotaur cum mixing with the lust draft. Your mind becomes consumed with the dual urges to fuck and be fucked. [EachCock] strains for a tight hole to fill while your [pussy] wants something big and drooling stuffed into its drooling opening. Even now you feel the wetness of your drooling loins salivating and you canÔÇÖt help but wonder if there is another minotaur around here somewhere to make a [name] sandwich.");
 	}
 	
 	outputText("\n\nYou drink your cocktail creation, each swallow sending a heady joy to your brain and [eachCock]. Your [face] twists into a silly grin as your loins stir with every sip.");
```

- "`[EachCock] {strain / strains}`" should be "`[EachCock] strains`" and
- `[playername]` should just be `[name]`

Perhaps you wanna file another PR to fix these?

**Comment by KiefKeith** [comment]
I looked again and found a couple more typos.  I'll get on that asap, thanks.

**Pull request merged**